keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -795,79 +840,149 @@ enhancement:
+### Monitoring requirements
 
 * **How can an operator determine if the feature is in use by workloads?**
 
+The feature is used if all of the following points apply:
+- `CSIDriver.spec.storageCapacity` is true for a CSI driver.
+- There are storage classes for that driver.
+- Volumes are using those storage classes.
+
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
 
+Pod status of the CSI driver deployment, existence of
+`CSIStorageCapacity` objects and ultimately the rate at which volumes
+are being created by the driver. The latter is exposed through the
+existing CSI sidecar metrics support.
+
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
 
+This depends on the workload and CSI driver. A very low rate of
+provisioned volumes per second and few total number of volumes overall
+may be fine for some workloads (long-running applications which use
+specialize storage like PMEM) while others may need a much higher rate
+(adding a local LVM scratch volume to every pod in the system).
+
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
 
-### Dependencies
+No.
 
-Will be added before the transition to beta.
+### Dependencies
 
 * **Does this feature depend on any specific services running in the cluster?**
 
-### Scalability
+No, just the ones that will also run without it enabled (apiserver,
+kube-scheduler, CSI driver).
 
-Will be added before the transition to beta.
+### Scalability
 
 * **Will enabling / using this feature result in any new API calls?**
 
+Yes.
+
+Enabling it in apiserver and CSI drivers will cause
+`CSIStorageCapacity` objects to be created respectively updated. The
+number of those objects is proportional to the number of storage
+classes and number of distinct storage topology segments. For
+centralized provisioning, the number of segments is probably low. For
+distributed provisioning, the each node where the driver runs
+represents one segment. The rate at which objects depends on how often
+storage usage changes.
+
+Enabling it in kube-scheduler will cause it to cache all
+`CSIStorageCapacity` objects via an informer.
+
 * **Will enabling / using this feature result in introducing new API types?**
 
+Yes, `CSIStorageCapacity`.
+
 * **Will enabling / using this feature result in any new calls to cloud
   provider?**
 
+A CSI driver might have to query the storage backend more often to be
+kept informed about available storage capacity.
+
 * **Will enabling / using this feature result in increasing size or count
   of the existing API objects?**
 
+No.
+
 * **Will enabling / using this feature result in increasing time taken by any
   operations covered by [existing SLIs/SLOs][]?**
 
+No. Scheduling of pods without volumes are not expected to be
+affected. Scheduling of pods with volumes is not covered by existing
+SLIs/SLOs.
```

Comment:
No SLO yet - but there is SLI:
https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md

Reply:
Thanks for the link, I rewrote the section.